contents/blog/sql-for-analytics.md

```diff
@@ -0,0 +1,133 @@
+---
+title: The basics of SQL for analytics
+date: 2023-09-06
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/hog_ql.png
+featuredImageType: full
+category: Engineering
+tags:
+ - Guides
+ - Explainers
+---
+
+One of SQL’s most important use cases is analytics. There are many ways to slice, modify, and interpret analytics data. SQL provides the depth and customizability to get insights from it, answer questions, and make data-driven decisions.
+
+This post goes over how to analyze your analytics data with SQL and recommendations for your analysis process.
+
+## The building blocks of SQL
+
+SQL provides complete customization into the retrival and formatting of analytics data. Because of the limitless options for doing this, it is useful to break down the analysis process into three main blocks: your analytics data, the SQL query, and the results.
```

Comment:
```suggestion
SQL provides complete customization into the retrieval and formatting of analytics data. Because of the limitless options for doing this, it is useful to break down the analysis process into three main blocks: your analytics data, the SQL query, and the results.
```